src/baldr/connectivity_map.cc

```diff
@@ -321,7 +211,7 @@ std::string connectivity_map_t::to_geojson(const uint32_t hierarchy_level) const
   std::unordered_map<size_t, polygon_t> boundaries;
   for (const auto& arity : arities) {
     auto& region = *regions.find(arity.second);
-    boundaries.emplace(arity.second, to_boundary(region, bbox->second.tiles));
+    boundaries.emplace(arity.second, to_boundary(region.second, bbox->second.tiles));
```

Comment:
`to_boundary` only used `region.second` so i've simplified it to only expect that. this makes `to_boundary` more generally useful as well.